modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/ApplicationThrottlingResetTestCase.java

```diff
@@ -0,0 +1,294 @@
+        APIOperationsDTO apiOperationsDTO = new APIOperationsDTO();
+        apiOperationsDTO.setVerb("POST");
+        apiOperationsDTO.setTarget("/*");
+        apiOperationsDTO.setAuthType("Application & Application User");
+        apiOperationsDTO.setThrottlingPolicy("Unlimited");
+        operations.add(apiOperationsDTO);
+        apiRequest.setOperationsDTOS(operations);
+
+        HttpResponse serviceResponse = restAPIPublisher.addAPI(apiRequest);
+        apiId = serviceResponse.getData();
+        // Create Revision and Deploy to Gateway
+        createAPIRevisionAndDeployUsingRest(apiId, restAPIPublisher);
+        restAPIPublisher.changeAPILifeCycleStatus(apiId, Constants.PUBLISHED);
+        waitForAPIDeploymentSync(user.getUserName(), APIName, APIVersion, APIMIntegrationConstants.IS_API_EXISTS);
+
+        // check backend
+        Map<String, String> requestHeaders = new HashMap<String, String>();
+        HttpResponse response = HttpRequestUtil.doGet(backendEP, requestHeaders);
+        Assert.assertEquals(response.getResponseCode(), 200, "Backend (dummy_api.xml) is not up and running");
+
+        // Create an Application for request count limit
+        requestCountApplicationDTO = restAPIStore.addApplication(applicationName1, policyName1, "",
+                "this-is-test-application-for-request-count-limit");
+
+        // Create an Application for bandwidth limit
+        bandwidthApplicationDTO = restAPIStore.addApplication(applicationName2, policyName2, "",
+                "this-is-test-application-for-bandwidth-limit");
+    }
+
+    @Test(groups = { "wso2.am" }, description = "Test reset application throttling policy with request count limit")
+    public void testResetPolicyWithRequestCountLimit() throws Exception {
+        //subscribe to API
+        SubscriptionDTO subscriptionDTO = restAPIStore.subscribeToAPI(apiId,
+                requestCountApplicationDTO.getApplicationId(), Constants.TIERS_UNLIMITED);
+        Assert.assertEquals(subscriptionDTO.getThrottlingPolicy(), Constants.TIERS_UNLIMITED,
+                "Error occurred " + "while subscribing to the api. Subscribed policy is not as expected as "
+                + Constants.TIERS_UNLIMITED);
+
+        // generate keys and token
+        ArrayList<String> grantTypes = new ArrayList<>();
+        grantTypes.add(APIMIntegrationConstants.GRANT_TYPE.CLIENT_CREDENTIAL);
+        ApplicationKeyDTO applicationKeyDTO = restAPIStore.generateKeys(requestCountApplicationDTO.getApplicationId(),
+                "3600", null, ApplicationKeyGenerateRequestDTO.KeyTypeEnum.PRODUCTION,
+                null, grantTypes);
+        String accessToken = applicationKeyDTO.getToken().getAccessToken();
+
+        Map<String, String> requestHeaders = new HashMap<String, String>();
+        requestHeaders.put("Authorization", "Bearer " + accessToken);
+        requestHeaders.put("accept", "text/xml");
+        requestHeaders.put("content-type", "application/json");
```

Comment:
There are already defined content type constants for the values of these content types

Reply:
Fixed with https://github.com/wso2/product-apim/pull/13493/commits/ea0b01ff6b639e1245e49754bb332e3fcf359753